doc/md_src_files/110_conclusion.md

```diff
@@ -3,6 +3,10 @@
 
 # Conclusion
 
-We presented a working hypothesis for how patch developers and patch appliers should prioritize their effort to mitigate different vulnerabilities. We have performed an initial pilot evaluation of the proposal and improved it, but the process we developed for evaluation is more important than the results. We invite further refinement of the prioritization mechanism. Further testing will be required before SSVC is ready for operational use. We endeavored to be transparent about our process and provide justification for design decisions.
+SSVC v2 presents a method for suppliers, deployers, and coordinators to use to prioritize their effort to mitigate different vulnerabilities.
+We have built on SSVC v1 through public presentation and feedback, private consultation, and continued analyst testing.
+The evaluation process we developed in v1 remains an important part of continued improvement of SSVC, and will be used to continue refinements of SSVC v2.
+We invite further refinement of the prioritization mechanism from the community as well, such as by [posting an issue](https://github.com/CERTCC/SSVC/issues).
```

Comment:
We invite further refinement

suggest: We invite participation and further refinement